### ISSUE TITLE
Facing issue in connecting with Google CloudSQL instance using CloudAuth proxy (running on GKE) from Liquibase Init container

### ISSUE THREAD

**Issue by sunnygoel8** [liquibase/docker#98]
Hi All,

We are trying to connect with Google CloudSQL instance using CloudAuth proxy from the liquibase init container but it;s not working. Liquibase container is continuously restarting and doesn't show anything in the logs. So it's getting difficult to troubleshoot the issue. 

Here's what we did. 
1.) Removed following line from Dockerfile and moved it in k8s Deployment yaml file as shown in Step 2.

> CMD ["sh", "-c", "docker-entrypoint.sh --url=${URL} --username=${USERNAME} --password=${PASSWORD} --classpath=/liquibase/changelog --changeLogFile=relative/changelog.xml update"]

2.) Please see below the InitContainers snippet from Deployment yaml file. 

datasource.url, datasource.username and datasource.password properties are defined in K8s secrets object. 

datasource.url property value uses Cloud Auth proxy service name (running on GKE cluster). We use that proxy in all our applications to securely connect with CloudSQL instance. 

>  initContainers:
>       - name: liquibase
>         image: <DockerImageName>
>         imagePullPolicy: IfNotPresent
>         envFrom:
>         - secretRef:
>             name: <SecretName>
>         command: ["/bin/sh"]
>         args: ["-c", "docker-entrypoint.sh --url=${datasource.url} --username=${datasource.username} --password=${datasource.password} --classpath=/liquibase/changelog --changeLogFile=db.changelog-master.yaml update"]

Since CloudAuth proxy is already referring to k8s secrets object to connectwith Cloud SQL instance, do we still need to pass those parameters in docker-entrypoint.sh command? Any help in this context will be appreciated.

**Comment by r2liquibase** [comment]
Yes. Liquibase running in this container requires those parameters. The container cannot access auth information on the host.
